server/controllers/groups.js

```diff
@@ -480,6 +487,15 @@ module.exports = function(app) {
                 }
               })
               .then(user => user || User.create(userAttr))
+              .then(user => {
+                return githubLib.fetchUser(contributor)
```

Comment:
can simplify to `.then(user => githubLib.fetchUser(contributor)).then(json => ...`